### PR TITLE
Update propsTable to have unique selectors for styles

### LINKED ISSUE #288: Global Props Table Markdown Styles Overwriting Table Styles
# Bug Report

## Description

A few styles were added to the props table for theme support. These styles aren't nested within a unique selector causing them to be global. 

https://github.com/cerner/terra-dev-site/blob/main/src/loader-components/PropsTable.module.scss#L20-L27

## Steps to Reproduce

1. Navigate to the terra-ui html-table [documentation site](https://engineering.cerner.com/terra-ui/components/terra-html-table/html-table/table).
2. Observe the table headers in each example have a white background
3. Inspect the `tr` elements to see a white background color is being applied by the props-table styles.

## Additional Context / Screenshots

I copy pasted the default example into a new application and observed the styles were inconsistent with the documentation site.

Project:

image

Terra UI:

image

## Expected Behavior

The Props Table styles should not be applied to any other element on the page.

## Possible Solution

Nest the styles under a selector or apply a class.

### PR DESCRIPTION
**Pull request by pranav300**
### Summary

This PR updates the unique selectors for the props Table so that the styles don't affect any other element on the page.

Updated `wdio` snapshots, due to changes in `prismjs@1.21.0`

Closes #288 

### Deployment Link

https://terra-dev-site-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
+  * Update propsTable to have unique selector for styles
```
Comment (StephenEsser) [inline CHANGELOG.md:6]: Because this is a bug fix it should be placed into a Fixed section

File: src/loader-components/PropsTable.module.scss [hunk 2]
```diff
@@ -17,12 +17,12 @@
...
   }
 
-  tr {
+  .tr {
     background-color: var(--terra-dev-site-props-table-tr-background-color, #fff);
     border-top: 1px solid #c6cbd1;
   }
 
-  tr:nth-child(2n) {
+  .tr:nth-child(2n) {
```
Comment (StephenEsser) [inline src/loader-components/PropsTable.module.scss:25]: A little confusing the class name is the tag name, but I double checked that this is the correct class being bound

Comment (mjhenkes) [reply src/loader-components/PropsTable.module.scss:25]: Thats probably my fault 😁

File: CHANGELOG.md (hunk 1 above)
Comment (pranav300) [reply CHANGELOG.md]: Updated here https://github.com/cerner/terra-dev-site/pull/289/commits/b61c4c573507e5437d665b178ab06b286449cd22

File: src/loader-components/PropsTable.module.scss (hunk 2 above)
Comment (emilyrohrbough) [reply src/loader-components/PropsTable.module.scss:25]: will this break table's screenshots once this is released?

**Pull request merged**